pkg/collector/corechecks/snmp/internal/devicecheck/devicecheck.go

```diff
@@ -150,6 +155,23 @@ func (d *DeviceCheck) Run(collectionTime time.Time) error {
 		d.sender.ReportMetrics(d.config.Metrics, values, tags)
 	}
 
+	// Get a system appropriate ping check
+	if d.config.PingEnabled {
+		log.Infof("ATTEMPTING TO RUN PING FOR HOST: %s, tags: %+v", d.config.IPAddress, tags)
```

Comment:
1/ I think we shouldn't use Info log level since this is expected behaviour. Is that for developement debugging purpose?

2/ Maybe we should avoid using UPPER CASE if not necessary.

Reply:
Yeah this was in here while I was doing local testing. I've removed it in favor of a trace log that's more appropriately formatted.